openedx/stanford/djangoapps/auth_lagunita/models.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models
+
+# Backwards compatible settings.AUTH_USER_MODEL
+USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
+
+
+class Info(models.Model):
+    """
+    This model contains extra fields that will be saved when a user registers.
+    """
+    user = models.OneToOneField(
+        USER_MODEL,
+    )
+    requested_marketing_optin = models.BooleanField(
+        default=False,
+    )
+    submitted_marketing_optin = models.BooleanField(
```

Comment:
Trying to understand the semantics: Does 'requested_marketing_optin' get set when the student checks the box on registration and 'submitted_marketing_optin' get set when the batch job sends it to Marketo? A couple of explanatory comments here would help.

Reply:
That is correct; I'll add a note.

Reply:
@dcadams Comments added to docstring above ^.